specification/security/resource-manager/Microsoft.Security/preview/2022-04-01-preview/examples/BasicQuery.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Basic Query",
+  "description": "A simple explorer query.",
+  "parameters": {
+    "api-version": "2022-04-01-preview",
+    "query": {
+      "scopes": [
+        "cfbbd179-59d2-4052-aa06-9270a38aa9d6"
```

Comment:
Is the example correct? the scope in the query is cfbbd179-59d2-4052-aa06-9270a38aa9d6, but the results are seemingly not scoped to it - what is the intent here?

Reply:
youre right, ill have it fixed (: